src/components/auth/Auth.js

```diff
@@ -0,0 +1,64 @@
+import auth0 from 'auth0-js';
+
+class Auth {
+  constructor() {
+    this.auth0 = new auth0.WebAuth({
+      // the following three lines MUST be updated
+      domain: 'coding-coach.eu.auth0.com',
+      audience: 'https://coding-coach.eu.auth0.com/userinfo',
+      clientID: 'bzQKiL4VQtjla1TeIgizh9YIoTyn01Zk',
+      redirectUri: 'http://localhost:3000/callback',
+      responseType: 'id_token',
+      scope: 'openid profile',
+    });
+
+    this.getProfile = this.getProfile.bind(this);
+    this.handleAuthentication = this.handleAuthentication.bind(this);
+    this.isAuthenticated = this.isAuthenticated.bind(this);
+    this.signIn = this.signIn.bind(this);
+    this.signOut = this.signOut.bind(this);
```

Comment:
I'm not 100% sure, but I think Babel handles this binding for us? I'm mobile and don't know if we use the class method autobind plugin.

Reply:
Disregard this. I wasn't aware that it wasn't a React class. The same rules might not apply.

Reply:
Use arrow functions for code consistency please